src/components/TodoItem/TodoItem.js

```diff
@@ -1,23 +1,36 @@
-import { FiMessageSquare } from "react-icons/fi";
-import { Item, Icons, Content } from "./TodoItem.styled";
+import { useState, useCallback, memo } from "react";
+import { Item, Icons, Content, ChatIcon } from "./TodoItem.styled";
 import TodoCheckbox from "../TodoCheckbox/TodoCheckbox";
 
-function TodoItem({ background, checkStatus, content }) {
+function TodoItem({ content, item, state, setState }) {
```

Comment:
component này cũng cần refactor lại để nhận vừa đủ những gì nó cần thôi Đức

component ko cần biết toàn bộ state của app, và không cần khả năng setState của toàn bộ state của app